src/herder/UpgradesTests.cpp

```diff
@@ -367,6 +367,19 @@ TEST_CASE("simulate upgrades", "[herder][upgrades]")
                               LedgerManager::GENESIS_LEDGER_BASE_FEE * 4,
                               LedgerManager::GENESIS_LEDGER_MAX_TX_SIZE * 4};
 
+    SECTION("oscillation")
```

Comment:
I imagine that you won't include this commit in the final PR as the next commit is making this irrelevant?